pkg/operator/ceph/config/monstore.go

```diff
@@ -102,7 +102,7 @@ func (m *MonStore) Set(who, option, value string) error {
 			"you may need to use the rook-config-override ConfigMap. output: %s", string(out))
 	}
 
-	logger.Infof("successfully set %q=%q=%q option to the mon configuration database", who, option, value)
+	logger.Infof("successfully set option %q (user %q) to the mon configuration database", who, option)
```

Comment:
```suggestion
logger.Infof("successfully set option %q (user %q) to the mon configuration database", option, who)
```

Need to interchange the option, and value for correct logging?

Reply:
thanks, fixed